app/project_type/docker.py

```diff
@@ -151,3 +151,8 @@ def _remove_file_system_unfriendly_characters(self, unescaped_path):
         """
         valid_chars = "-_.()%s%s" % (string.ascii_letters, string.digits)
         return ''.join(c for c in unescaped_path if c in valid_chars)
+
+    def project_id(self):
+        # TODO: Ideally each docker container could have a unique project_id since we can fetch/atomize any docker
+        # containers in parallel without conflicts
+        return self._image
```

Comment:
For Docker, I don't think we can fetch multiple images concurrently for the single docker daemon that may be running.

I'm thinking the project_id for Docker in this design should return a static 'docker' or something with a comment indicating the reason for the static-ness.